server/startup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+PORT=3000
+
+echo pwd
+cd core
+flutter pub get
+
+cd ../model
+flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs
+
+cd ..
+flutter pub get && flutter pub run intl_generator:extract_to_arb --output-dir=./lib/l10n lib/main/localizations/app_localizations.dart
+
+flutter pub get && flutter pub run intl_generator:generate_from_arb --output-dir=lib/l10n --no-use-deferred-loading lib/main/localizations/app_localizations.dart lib/l10n/intl*.arb
+
+flutter build web --dart-define=SERVER_URL='https://dev.open-paas.org'
+
+echo 'preparing port' $PORT '...'
+fuser -k 3000/tcp
+
+cd build/web/
+
+# Start the server
+echo 'Server starting on port' $PORT '...'
+python3 -m http.server $PORT
```

Comment:
urg python, really? T_T

Reply:
please help me fix up :(

Reply:
If we are just serving static content we may consider something like nginx

Reply:
Me it's more like I hate Python :D After it depends how flutter build this for webapp content, I have no idea

Reply:
For the web server, no constrain with `flutter build`. I chose `python3` because it is in tutorial, that's all. Please guide me the better option for our application.

Reply:
https://docs.flutter.dev/deployment/web#building-the-app-for-release that's the tuto right? Yeah i'm rusty in frontend webapp but let's not use python :) let's check something else

Reply:
https://rebar-ahmad.medium.com/env-aware-flutter-apps-in-docker-b899d0deebfd

The same but with nginx :)